tox.ini

```diff
@@ -18,7 +18,7 @@ envlist =
 
 [testenv]
 commands =
-    {env:_PYTEST_TOX_COVERAGE_RUN:} pytest --lsof
+    {env:_PYTEST_TOX_COVERAGE_RUN:} pytest --lsof {posargs}
```

Comment:
Why do we need `{posargs}` here?  Left over from local testing?

Reply:
@Zac-HD without posargs its impossible to run specific tests in calls like `tox -- testing/test_some.py` since it wouldnt pass on